plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -301,36 +319,57 @@ static bool update_enhancements(obs_data_t *settings)
 	}
 
 	if (update) {
-		bool enabled = (mbbrc && cqm);
+		bool enabled = (cqm);
```

Comment:
As far as I understand, this is meant to keep Subjective Video Enhancements disabled if a user just upgraded from a version of OBS that had MBBRC and CQM settings set and they had previously disabled either MBBRC or CQM, while erasing both old settings. This can probably be left as-is.

Reply:
I've made this change and force pushed.